libs/sdk-common/Cargo.toml

```diff
@@ -22,7 +22,12 @@ serde = { workspace = true }
 serde_json = { workspace = true }
 strum_macros = { workspace = true }
 thiserror = { workspace = true }
-tonic = { workspace = true }
+tonic = { workspace = true, features = [
+    "tls",
+    "transport",
+    "tls-roots",
+    "tls-webpki-roots",
+] }
 
```

Comment:
What was the requirement to add these features?

Reply:
Without these, gRPC calls to the Breez Server failed.

Looks like `tonic` doesn't include basic tls features in its default setup, so they have to be added explicitly. We have these extra features in Breez SDK too, I assume for the same reason.